src/HttpHeaders.php

```diff
@@ -123,16 +123,27 @@ public function send()
         }
     }
 
+    /**
+     * @return array
+     */
+    public function getSentHeaders()
+    {
+        /** @link http://php.net/manual/ru/function.headers-list.php#120539 */
+        return (PHP_SAPI === 'cli' && \function_exists('xdebug_get_headers'))
+            ? xdebug_get_headers()
+            : headers_list();
+    }
+
     /**
      * Sends HTTP Header
      *
-     * @param string      $name               Header name
-     * @param string|null $value              Header value
-     * @param int         $http_response_code HTTP response code
+     * @param string      $name             Header name
```

Comment:
let's not change spaces

Reply:
They were aligned coz the method argument was renamed to comply with PSR-2.